contracts/utils/CloneFactory.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity ^0.4.24;
+
+/*
+The MIT License (MIT)
+
+Copyright (c) 2018 Murray Software, LLC.
```

Comment:
:)

Reply:
Hey :) apologise.. I thought if I reuse someone code with such copyright then it should be put as well. Shouldn't it ?

Reply:
Haha my bad, my comment was a bit cryptic. You're indeed correct, I was just glad you were giving proper attribution to the author, as opposed to simply copying over their code :)